kalite/main/content_rating_models.py

```diff
@@ -5,14 +6,18 @@
 from securesync.models import DeferredCountSyncedModel
 
 class ContentRating(DeferredCountSyncedModel):
+
+    minversion = "0.15.0"
+
     class Meta:
         unique_together = ("content_kind", "content_id", "user")
 
     # Maintain info on content type -- could be video, exercise, etc, but we should be able to uniquely id it
     content_kind = models.CharField(max_length=100, db_index=True, blank=False)
     content_id = models.CharField(max_length=100, db_index=True, blank=False)
+    content_source = models.CharField(max_length=100, db_index=True, default=settings.CHANNEL)

```

Comment:
They can actually be combined manually yes, just copy-paste the migration into where it fits. Admittedly, it may become a mess and perhaps worth putting separately in case it causes issues. AFAIK data and schemamigrations are made of the same and interpreted the same way by South.